src/coreclr/jit/emitarm64.cpp

```diff
@@ -1124,6 +1124,17 @@ void emitter::emitInsSanityCheck(instrDesc* id)
             assert(id->idInsOpt() == INS_OPTS_SCALABLE_H);
             break;
 
+        case IF_SVE_GD_2A: // .........x.xx... ......nnnnnddddd -- SVE2 saturating extract narrow
+            elemsize = id->idOpSize();
+            assert(insOptsScalable(id->idInsOpt()));
+            assert(isVectorRegister(id->idReg1())); // nnnnn
+            assert(isVectorRegister(id->idReg2())); // ddddd
+            assert(optGetSveElemsize(id->idInsOpt()) != EA_8BYTE);
+            assert(isValidVectorElemsize(optGetSveElemsize(id->idInsOpt()))); // xx
```

Comment:
I don't we need a `isValidVectorElemsize()` check - the  `insOptsScalable()` check will only allow valid scalable vector types.

Reply:
@kunalspathak for this one as it comes from the boilerplate code. Should the boilerplate of this be replaced with `assert(isScalableVectorSize(elemsize));` ?

Reply:
I'll fix this up in upcoming PRs.